pkg/security/agent/agent.go

```diff
@@ -41,7 +41,7 @@ type RuntimeSecurityAgent struct {
 	cancel               context.CancelFunc
 
 	// activity dump
-	storage *activitydump.ActivityDumpStorageManager
+	storage *activitydump2.ActivityDumpStorageManager
```

Comment:
can't you use the `dump` package name here?

Reply:
Rrrrrr ... my IDE automatically does this, updating ! Thanks !